plugins/GSdx/GSTextureCache.cpp

```diff
@@ -517,27 +517,27 @@ GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, int
 		//
 		// From a performance point of view, it might cost a little on big upscaling
 		// but normally few RT are miss so it must remain reasonable.
-		if (s_IS_OPENGL) {
-			if (m_preload_frame && TEX0.TBW > 0) {
-				GL_INS("Preloading the RT DATA");
-				// RT doesn't have height but if we use a too big value, we will read outside of the GS memory.
-				int page0 = TEX0.TBP0 >> 5;
-				int max_page = (MAX_PAGES - page0);
-				int max_h = 32 * max_page / TEX0.TBW;
-				// h is likely smaller than w (true most of the time). Reduce the upload size (speed)
-				max_h = std::min<int>(max_h, TEX0.TBW * 64);
-
-				dst->m_dirty.push_back(GSDirtyRect(GSVector4i(0, 0, TEX0.TBW * 64, max_h), TEX0.PSM));
-				dst->Update();
-			} else {
+		if (m_preload_frame && TEX0.TBW > 0) {
```

Comment:
So I think it will be better to do
```
bool supported_fmt = m_can_convert_depth || psm_s.depth == 0;
if (m_preload_frame && TEX0.TBW > 0 && supported_fmt) {
```
It might need some tuning to compile.

Reply:
What about s_IS_OPENGL in the switch case for the debug?

Reply:
Sorry for the delay. No don't add it. `ENABLE_OGL_DEBUG` was initially an helper for the gl layer but it spread everywhere. It can help Dx debug too.

Reply:
Gotcha. Updated the code.